### ISSUE TITLE
Python 3.10 pip installation error

### ISSUE THREAD

**Issue by Tilix4** [AcademySoftwareFoundation/OpenTimelineIO#1384]
## To Reproduce

1. Windows 10 & OSX 11.6.3
2. Python 3.10.2
3. `pip install opentimelineio`

## Expected Behavior

Correct pip installation.

## Logs

[Erreur_installation_opentimelineio.txt](https://github.com/AcademySoftwareFoundation/OpenTimelineIO/files/9382341/Erreur_installation_opentimelineio.txt)

## Additional Context

Installation in Blender's python for https://gitlab.com/superprod/stax_suite/vse_io/

**Comment by meshula** [comment]
something strange going on with multiple python installations? from the log we can see a reference to 3.10

```
 copying contrib\opentimelineio_contrib\adapters\contrib_adapters.plugin_manifest.json -> build\lib.win-amd64-cpython-310\opentimelineio_contrib\adapters
 ```
and then
```
  RuntimeError: CMake Error at C:/Users/Hayle/AppData/Local/Temp/pip-build-env-jiu3fz2u/overlay/Lib/site-packages/cmake/data/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
    Could NOT find Python (missing: Interpreter) (found version "2.7.9")
```

**Comment by Tilix4** [comment]
It works well for Blender versions embedding Python 3.9 (Blender up to 3.0)

**Comment by reinecke** [comment]
This should be fixed by #1256. Duplicate of #1224.

This should be available on PyPI in the next release, in the meantime you can pull a development build from a recent github action execution.
